sbncode/Metadata/FileCatalogMetadataSBN_service.cc

```diff
@@ -27,7 +28,15 @@ FileCatalogMetadataSBN(fhicl::ParameterSet const& pset, art::ActivityRegistry& r
   fProjectSoftware = pset.get<std::string>("ProjectSoftware","");    
   fProductionName = pset.get<std::string>("ProductionName","");  //Leave as default value if not running a production   
   fProductionType = pset.get<std::string>("ProductionType",""); //Leave as default value if not running a production
+  fMerge = pset.get<int>("Merge", -1);
+  fParameters = pset.get<std::vector<std::string> >("Parameters", std::vector<std::string>());
```

Comment:
And have parameters represented in the fcl file as nested vectors `[[key1, value1], [key2, value2], ...`

Reply:
Or a fcl table.